src/transformers/models/cvt/configuration_cvt.py

```diff
@@ -0,0 +1,159 @@
+# coding=utf-8
+# Copyright 2022 AnugunjNaman and The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
```
Sorry, we usually don't do this, your name is included in the documentation page ("this model was contributed by...").